Lighting Tweaks

chefferz opened the pull request:
Tweaks lighting wattage to be far more realistic to futuristic tech.

1) Lighting now takes half the wattage as normal, A standard light tube draws only 80w now, vs 160w. 
2) The emergency lighting color is now fire engine/emergency red
3) Night shift lighting is now slightly brighter to ensure hallways aren't near pitch black in some areas.

chefferz committed:
Merge pull request #2 from Citadel-Station-13/master

Update

chefferz committed:
Lighting Update

Fire Engine Red Emergency
Slightly Brighter Nighttime
Less power usage

chefferz committed:
Update lighting.dm

deathride58 commented on code/modules/power/lighting.dm:
```diff
@@ -214,13 +214,13 @@
 	var/nightshift_allowed = TRUE	//Set to FALSE to never let this light get switched to night mode.
 	var/nightshift_brightness = 8
-	var/nightshift_light_power = 0.45
+	var/nightshift_light_power = 0.70
```
This is way too close to the baseline light power (which is 0.75). With this proposed value, it'll be impossible to actually notice any difference between baseline lighting and night shift lighting, and there'd no longer be any mechanical advantages/disadvantages to night shifts (such as, but not limited to, nightmares getting some wiggle room with station lighting, podpeople needing to stay closer to station lights, botany plants growing slower with roundstart light setups, etc etc). This proposed value practically removes night shifts.

chefferz committed:
Update lighting.dm

chefferz committed:
Merge pull request #3 from Citadel-Station-13/master

update 2

chefferz committed:
Doubles stamina regen when standing/laying down


Does just as it says on the tin, doubles stamina regeneration when standing/laying down. Keeps hard stam crit regen as buffing that would make it harder/difficult to cuff. 

This PR is going to have to be playtested as it's not particularly easy to locally test. Testmerging and tweaking it w/ observed input would be preferred.
